packages/datetime/test/dateTimePickerTests.tsx

```diff
@@ -8,7 +8,6 @@
 import { assert } from "chai";
 import { mount } from "enzyme";
 import * as React from "react";
-
```

Comment:
Let's keep this newline in place to differentiate external dependencies from internal dependencies.